src/Map/Map.router.js

```diff
@@ -34,6 +31,7 @@ export class MapRouter {
       this.handleRequest(this.getParcelPNG)
     )
     // TODO: add an endpoint for Estates someday 🏌🏼‍
```

Comment:
I think we can delete the TODO! ⛳️

Reply:
It's being deleted in another PR, because it refers to a `/estate` endpoint https://github.com/decentraland/marketplace/pull/265/files#diff-32768cada300df64cc722464f0ad95fbR30